force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -69,7 +74,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The quotient of the division, or 0 if the denominator is zero.
      */
     public static Decimal division(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        if(b==0){
```

Comment:
Spacing these out helps for better readability ' if(b == 0) ', what do you think?